src/app/data/scenario.service.ts

```diff
@@ -131,9 +131,16 @@ export class ScenarioService {
   }
 
   public create(s: Scenario) {
+    s.steps.forEach((st: Step) => {
+      st.title = utoa(st.title);
+      st.content = utoa(st.content);
+    });
+
     var params = new HttpParams()
```

Comment:
```suggestion
    const params = new HttpParams()
```

We have some var keywords left in our code and should remove them at least in the files that come with this PR :) 
We should prefer the block scoped key words let and const over the var key word.

Reply:
I will look after that.

Reply:
well, I changed all the var that I can find on my push, if you notice more just let me know.